test/fixtures.rb

```diff
@@ -40,6 +40,10 @@ class Application < Hanami::Application
 
       default_response_format :html
 
+      security.x_frame_options "DENY"
+      security.content_security_policy "form-action 'self'; referrer origin-when-cross-origin; reflected-xss block; manifest-src 'self'; frame-ancestors 'self'; base-uri 'self'; default-src 'none'; connect-src 'self'; img-src 'self'; style-src 'self'; font-src 'self'; object-src 'self'; plugin-types application/pdf application/x-shockwave-flash application/x-java-applet; child-src 'self'; media-src 'self'"
```

Comment:
@joneslee85 Every time I used CSPRO was for projects already in production since years. That header is useful to collect data for a long period of time and then plan the switch off with CSP.

That represent a soft transition from an in browser warning that you can collect to a browser exception that stops the page to work as usual.

Do we have this legacy problem with Hanami?

The idea of introducing CSP in the early versions of the framework, was to avoid that painful transition. I think we can safely remove CSPRO support.

In case someone needs it, they can always use: 1. Default headers 2. a shared module for each action (hint `controller.prepare`) and always send that report-only header.

What do you think?